presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBucketedTables.java

```diff
@@ -77,10 +77,10 @@ private static void populateDataToHiveTable(String destination, String source, S
     {
         String queryStatement = format("INSERT INTO TABLE %s PARTITION (%s) SELECT * FROM %s", destination, partition, source);
 
-        onHive().executeQuery("SET mapreduce.map.memory.mb=512");
-        onHive().executeQuery("SET mapreduce.map.java.opts=-Xmx384m");
-        onHive().executeQuery("SET mapreduce.reduce.memory.mb=512");
-        onHive().executeQuery("SET mapreduce.reduce.java.opts=-Xmx384m");
+        onHive().executeQuery("SET mapreduce.map.memory.mb=256");
```

Comment:
Is there guarantee that every `onHive().executeQuery(...)` will execute in the same session/connection?
API doesn't seem explicit about that... And otherwise these session config statements will be void.

Reply:
They work. The same connection will be provided. Though, I do not know how sessions work in hive. Anyway, I had a mistake with `Xmx` set to `3844m`, that caused all hive queries to fail.